test/functionalTest/cases/715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test

```diff
@@ -238,92 +238,96 @@ Date: REGEX(.*)
 06. Dump accumulator to see X-Auth headers (or not)
```

Comment:
It seems that in some existing .test, the notifications sent to accumulator has changed from XML to JSON. However, I  understand that XML was the default format. Thus, what have happened with these tests?